### PR TITLE
Workbench

### PR DESCRIPTION
**Pull request by birm**
Placeholder PR, for now.

We'll want to bring in the GSOC code when it's ready.

### CODE REVIEW

**Comment by birm** [comment]
@akhil-rana  let me know when you're ready for us to merge this PR :)

File: apps/landing/landing.html [hunk 1]
```diff
@@ -50,6 +50,12 @@
             <li class="nav-item  link" style="font-family: sans-serif;">
                 <a class="nav-link" href="../table.html"> <i class="fas fa-list-ul"></i>  Slides</a>
             </li>
+            <li class="nav-item link">
```
Comment (akhil-rana) [inline apps/landing/landing.html:53]: We're now having the `info` tab in NavBar in landing.html, table.html, info.html which was removed from navbar in #376

**Comment by akhil-rana** [comment]
And also I'm not sure if the footer should be like this when there are less number of slides.

DeepinScreenshot_select-area_20200830030504

**Comment by nanli-emory** [comment]
> And also I'm not sure if the footer should be like this when there are less number of slides.
> 
> DeepinScreenshot_select-area_20200830030504

when the screen width is small, the footer becomes this way. It should be the data table things.

**Comment by akhil-rana** [comment]
Do you think we should remove the `info` option from the navbar? Since it is now present at multiple places.

**Comment by nanli-emory** [comment]
> Since



> Do you think we should remove the `info` option from the navbar? Since it is now present at multiple places.

Don't worry about the `info` option. We can keep it ~~~~ 💯

**Comment by akhil-rana** [comment]
Okay. Great. 
Thank you.

**Comment by akhil-rana** [comment]
I think we should be ready to merge into develop now.

**Pull request merged**